modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -65,6 +66,9 @@ GrainTracker::GrainTracker(const InputParameters & parameters)
     _remap(getParam<bool>("remap_grains")),
     _nl(_fe_problem.getNonlinearSystemBase()),
     _feature_sets_old(declareRestartableData<std::vector<FeatureData>>("unique_grains")),
+    _poly_ic_uo(parameters.isParamValid("polycrystal_ic_uo")
```

Comment:
Here I optionally couple in the same `PolycrystalUserObject` which I can replicate data from.